WinUIGallery/Samples/ControlPages/Design/IconographyPage.xaml.cs

```diff
@@ -49,6 +49,7 @@ public IconographyPage()
     {
         // Fill filtered items
         this.InitializeComponent();
+        this.SuspendOuterScrolling();
```

Comment:
I would really prefer this to be a separate PR, especially since I would like to discuss if we can do this differently.